# Disables PulseAudio's update-map-file target because it causes configure errors on Linux.

--- a/src/pulse/meson.build
+++ b/src/pulse/meson.build
@@ -70,7 +70,7 @@ if glib_dep.found()
   libpulse_headers += 'glib-mainloop.h'
 endif
 
-if host_machine.system() != 'windows' and host_machine.system() != 'darwin'
+if false
   run_target('update-map-file',
     command : [ join_paths(meson.source_root(), 'scripts/generate-map-file.sh'), 'map-file', 'libpulse.def',
                 [ libpulse_headers, 'simple.h', join_paths(meson.build_root(), 'src', 'pulse', 'version.h') ] ])
